### PR TITLE
Feature/out folder name

### PR DESCRIPTION
**Pull request by byb121**
added an output file called 'rg_file_names.txt', which has the name of the output interleaved fastq file, but it becomes useless if the workflow is run by Dockstore, as it renames the final outputs.

### CODE REVIEW

**Comment by keiranmraine** [comment]
It will still be useful for confirming the expected number of files so not completely useless..

File: README.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ These workflows were built based on existing tools below:
 
 #### Inputs
 
-* `fastq_in` - An interleaved FastQ file.
+* `fastq_in` - An gzipped interleaved FastQ file.
```
Comment (drjsanger) [inline README.md]:
```suggestion
* `fastq_in` - A gzipped interleaved FastQ file.
```

File: README.md [hunk 2]
```diff
@@ -32,12 +32,13 @@ These workflows were built based on existing tools below:
 * `chksum_post_server_response` - POST server response in a text file. Optional, if no `post_address` is given.
 * `interleave_report_json`- A json report evaluating the analysis of the `fastq_in`.
 * `interleave_ifastq_out` - Interleaved gzipped FastQ output file.
+* `rg_file_names` - A file which has the file name of the gzipped FastQ output file of the workflow. **NOTE:** The file will be useless if the workflow is ran by Dockstore, as the output file will be renamed by Dockstore.
```
Comment (drjsanger) [inline README.md]:
```suggestion
* `rg_file_names` - A file which has the file name of the gzipped FastQ output file of the workflow. **NOTE:** The file will be useless if the workflow is run by Dockstore, as the output file will be renamed by Dockstore.
```

File: README.md [hunk 3]
```diff
@@ -32,12 +32,13 @@ These workflows were built based on existing tools below:
...
 
 ### chksum_seqval_wf_paired_fq.cwl
 
 #### Inputs
 
-* `fastq_in` - A **list** of two FastQ files.
+* `fastq_in` - A **list** of two paired gzipped FastQ files.
 * `post_address` - A **list** of two URLs to send checksum results of coresponding FasetQ to via POST. It's required and **NOT** optional, but two empty strings can be used if no POST is required.
```
Comment (drjsanger) [inline README.md]:
```suggestion
* `post_address` - A **list** of two URLs to send checksum results of corresponding FastQ to via POST. It's required and **NOT** optional, but two empty strings can be used if no POST is required.
```

File: README.md [hunk 4]
```diff
@@ -47,6 +48,7 @@ These workflows were built based on existing tools below:
 * `chksum_post_server_response` - A list of two text files. Each has POST server response of one of the `fastq_in`. Optional if empty `post_address` was used.
 * `interleave_report_json`- A json report evaluating the analysis of the `fastq_in`.
 * `interleave_ifastq_out` - Interleaved gzipped FastQ output file.
+* `rg_file_names` - A file which has the file name of the gzipped FastQ output file of the workflow. **NOTE:** The file will be useless if the workflow is ran by Dockstore, as the output file will be renamed by Dockstore.
```
Comment (drjsanger) [inline README.md]:
```suggestion
* `rg_file_names` - A file which has the file name of the gzipped FastQ output file of the workflow. **NOTE:** The file will be useless if the workflow is run by Dockstore, as the output file will be renamed by Dockstore.
```

File: cwls/chksum_seqval_wf_interleaved_fq.cwl [hunk 5]
```diff
@@ -66,15 +71,23 @@ steps:
       post_headers:
         source: post_headers
     out: [chksum_json, post_server_response]
-    run: cgp-chksum.cwl
+    run: https://raw.githubusercontent.com/cancerit/dockstore-cgp-chksum/0.2.0/Dockstore.cwl
```
Comment (drjsanger) [inline cwls/chksum_seqval_wf_interleaved_fq.cwl]: Should this be linked to a version or not?

File: cwls/chksum_seqval_wf_paired_fq.cwl [hunk 6]
```diff
@@ -79,14 +84,22 @@ steps:
     out: [chksum_json, post_server_response]
     scatter: [in_file, post_address]
     scatterMethod: dotproduct
-    run: cgp-chksum.cwl
+    run: https://raw.githubusercontent.com/cancerit/dockstore-cgp-chksum/0.2.0/Dockstore.cwl
```
Comment (drjsanger) [inline cwls/chksum_seqval_wf_paired_fq.cwl]: As previous query about linking to a specific version

File: cwls/chksum_seqval_wf_interleaved_fq.cwl (hunk 5 above)
Comment (byb121) [reply cwls/chksum_seqval_wf_interleaved_fq.cwl]: Yes, but I thought "/0.2.0/" part was referring a version.  How can I link it to a version if that's not the correct path?

Comment (drjsanger) [reply cwls/chksum_seqval_wf_interleaved_fq.cwl]: As discussed, the two comments about version number can be ignored. This way is the lesser of two evils.

**Comment by byb121** [comment]
@drjsanger corrected those typos, please merge if you're happy with the changes.

**Pull request merged**